fe/fe-core/src/main/java/com/starrocks/clone/BackendLoadStatistic.java

```diff
@@ -89,17 +89,24 @@ public int compare(BackendLoadStatistic o1, BackendLoadStatistic o2) {
     }
 
     public static class BeStatComparatorForUsedPercent implements Comparator<BackendLoadStatistic> {
-        private TStorageMedium medium;
+        private final TStorageMedium medium;
+        private final boolean isAscending;
 
         public BeStatComparatorForUsedPercent(TStorageMedium medium) {
             this.medium = medium;
+            this.isAscending = true;
+        }
+
+        public BeStatComparatorForUsedPercent(TStorageMedium medium, boolean isAscending) {
```

Comment:
Now disk balance, it's the tablet balance. It's better to modify the title.

Reply:
done